### ISSUE TITLE
add reference data

### ISSUE THREAD

**Issue by aremazeilles** [aremazeilles/beat_routine#2]
We need reference data, with expected result, to make sure any code evolution does not affect the computed results.

**Comment by aremazeilles** [comment]
@juritaborri, culd you share with me some reference input - output data, so that we can add them to the repository?

**Comment by juritaborri** [comment]
Due to the COVID emergency, we have not available real data, I have tested the code with similuated data without any biomechanical sense, is it ok used them for now?

**Comment by aremazeilles** [comment]
We need some reference code, to make sure that any change we do to the code does not affect the output. 
So we would need reference input and output data. 
When you mention these simulated data, I assume the output (PI score) makes sense right?
To start with, I think that simulated data is fine.

We are working on a function, computing a score, i.e y = f(x). We need x and y samples, to make sure function f remains correct upon changes we apply. Mathematically speaking, if x and y are from real experiments or simulated ones, it does not change anything. 

You can have a look at the octave exmaple, https://github.com/eurobench/pi_octave_csic/tree/master/tests/data

We store input and associated output files. We would need similar association input-output

**Comment by aremazeilles** [comment]
I would wait for having access to this reference data (input-output) to continue the testing of your code. Just ping me when you have it, through github or through email.

**Comment by aremazeilles** [comment]
So here are the operations I conducted, let's see if you can do so for the next time by yourself.
I will put the git terminal command. There exists several different GUI for using GIT. But I prefer terminal commands.
This is quite direct under linux, but you can do very similar stuff under Windows, using the [Git for Windows tool](https://gitforwindows.org/).

Assuming we are located in the root of the repository, go to the master branch: 

```shell
git checkout master
# create a branch for making the changes
git checkout -b input-output
# place in the branch all your changes
# check changes wrt last code present
git status
# check in detail the differences
git diff 
# commit your changes. Note that new files and directory hould be added first (check git status output)
git add input output
git commit -am "adding reference data, and adjust code"
# push the changes to the server. 
# As it is a new branch, the command is slightly more complex
 git push --set-upstream origin input-output
# next time, a git push would be sufficient
```
Now the branch is on the github server.

**Comment by aremazeilles** [comment]
Looking at the repo page, I can see the upoaded branch : 
![github-screenshot](https://user-images.githubusercontent.com/12545372/79304603-06d48d80-7ef2-11ea-9f30-3f9e5b139124.png)

Click on the button `Compare and Pull request`, and create the pull request.
You should see the following:

![github2](https://user-images.githubusercontent.com/12545372/79304782-5f0b8f80-7ef2-11ea-9131-44b075294ee8.png)

From there, we can associate it to the relate issue, by adding in the write are "Fix #2". On the right, you can indicate this is still in progress, and assign it to me.

**Comment by aremazeilles** [comment]
From there, we can start discussing, on the code evolution (which is what we will do). And we always have the possibility to look at the code evolution, with the upper button "Files changed".
